src/app/screens/DetailArticle/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { View, Image, TouchableOpacity, Text } from 'react-native';
+import CustomText from '@components/CustomText';
+import { Navigation } from '@interfaces/navigation';
+import { formatDate } from '@utils/dateUtils';
+import icAddInactive from '@assets/TabBar/icAddpostInactive.png';
+import icAddActive from '@assets/TabBar/icAddpostActive.png';
+import icFavouriteInactive from '@assets/TabBar/icFavoriteInactive.png';
+import icFavouriteActive from '@assets/TabBar/icFavoriteActive.png';
+
+import icDefaultArticleImage from '../Home/components/ArticleItem/assets/default_article_image.jpg';
```

Comment:
Move default_article_image.jpg to assets and use '@assets/defaultArticleImage.jpg